frontend/openchat-client/src/stores/chat.ts

```diff
@@ -371,6 +375,16 @@ export const selectedChatStore = derived(
     },
 );
 
+export const selectedCommunityChatStore = derived(
```

Comment:
This is not necessary. You can just user `chatSummariesStore` or `chatSummariesListStore` - this will already be filtered to the selected scope (whether it is groups, direct or a selected community).